src/styles/layouts/_main.scss

```diff
@@ -1,23 +1,25 @@
 .l-main {
-  // max-width: $main-max-width;
-  // margin: 0 auto;
   background: white;
   box-shadow: 0 0 5px rgba(black, 0.1);
+  padding-top: $size-header;
 }
 
 .l-header {
-  position: relative;
+  position: absolute;
   z-index: 2;
+  top: 0;
+  left: 0;
 }
 
 .l-content {
   position: relative;
   z-index: 1;
   overflow: hidden;
+}
 
-  &.-fullscreen {
-    height: calc(100vh - 60px);
-  }
+// Util class, must override height property value
+.-fullscreen {
+  height: calc(100vh - #{$size-header}) !important;
```

Comment:
mmmm.... I don't like this... Why are we using important? I think that I understand the reason but we should avoid this kind of things